user_guide_src/source/libraries/caching.rst

```diff
@@ -59,9 +59,9 @@ more complex, multi-server setups.
 If you have more than one application using the same cache storage, you can add a custom prefix
 here that is prepended to all key names.
 
-**$path**
+**$file**
 
-This is used by the ``file`` handler to show where it should save the cache files to.
+This is an array of settings specific to the  ``File`` handler determine how it should save the cache files.
```

Comment:
Some word is missing here.
Maybe ..._handler **to** determine_...?

Reply:
Yep! Good catch, thank you.